src/main/java/core/basesyntax/cvswork/FileWrittenImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.cvswork;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class FileWrittenImpl implements FileWriter {
+    @Override
+    public void write(String filePath) {
+        if (new File(filePath).exists()) {
```

Comment:
let's remove this if else

Reply:
but this operation checks if this file exists
since BufferedReader creates a file in the named path instead of creating a RuntimeException

Reply:
> let's remove this if else
>but this operation checks if this file exists
since BufferedReader creates a file in the named path instead of creating a RuntimeException